src/util/valuetransformer.cpp

```diff
@@ -51,6 +51,13 @@ ValueTransformer* ValueTransformer::parseFromXml(QDomElement transformElement,
             }
         } else if (element.nodeName() == "Not") {
             pTransformer->addTransformer(new TransformNot());
+        } else if (element.nodeName() == "Match") {
```

Comment:
Great: it fixes bug https://bugs.launchpad.net/mixxx/+bug/1376277
I would prefer to use "Equals" because "Match" can be also used for counter parts.